### ISSUE TITLE
Some code errors  in TAG

### ISSUE THREAD

**Issue by lyy-zz** [yjxiong/action-detection#85]
Anthor use 8 gpus to training, i only 2,so i set the argument to `-b 4 -i 4`, but no matter how i adjust the lr strategy ,the model can't converge. I am very confused until i check the `iter_size`  code, find a IndentationError leads to logic errors, in https://github.com/yjxiong/action-detection/blob/e88344b2c88df28a8794fd6fce5cf544f52f1e9e/binary_train.py#L185-L201 

Should be like https://github.com/yjxiong/action-detection/blob/e88344b2c88df28a8794fd6fce5cf544f52f1e9e/ssn_train.py#L238-L254

I have some idea to change this part of the code,maybe not well
1. avoid i=0，i % args.iter_size == 0
2. adapt len(train_loader)% args.iter_size != 0，instead of ignore remainder
```python
        if (i+1) % args.iter_size == 0 or (i+1)==len(train_loader):
            # scale down gradients when iter size is functioning
            ......
                    for p in g['params']:
                        if (i+1)==len(train_loader) and len(train_loader)% args.iter_size != 0:
                            p.grad /= len(train_loader)% args.iter_size
                        else:
                            p.grad /= args.iter_size

            ......

            optimizer.step()
            optimizer.zero_grad()
```
https://github.com/yjxiong/action-detection/blob/e88344b2c88df28a8794fd6fce5cf544f52f1e9e/load_binary_score.py#L146
Should be
```python
            self.bg_pool.extend([(v.id, prop) for prop in v.get_bg(self.bg_iou_thresh)])
```
Have other tiny errors,about print infomation as follows:
https://github.com/yjxiong/action-detection/blob/e88344b2c88df28a8794fd6fce5cf544f52f1e9e/binary_train.py#L117
Should be
```python
        print(('group: {} has {} params, lr_mult: {}, decay_mult: {}'.format(
```

https://github.com/yjxiong/action-detection/blob/e88344b2c88df28a8794fd6fce5cf544f52f1e9e/binary_train.py#L251
Should be
```python
                  'Time {batch_time.val:.4f} ({batch_time.avg:.4f})\t'
```

**Comment by yjxiong** [comment]
Thanks! Please make a PR, I will review and merge it to the codebase.

**Comment by yjxiong** [comment]
Resolved by #88
